cypress/integration/requests/patron-initiated/request-item-with-fixed-loan-recall.spec.js

```diff
@@ -0,0 +1,144 @@
+import moment from 'moment';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import generateItemBarcode from '../../../support/utils/generateItemBarcode';
+import DevTeams from '../../../support/dictionary/devTeams';
+import TestTypes from '../../../support/dictionary/testTypes';
+import TopMenu from '../../../support/fragments/topMenu';
+import RequestPolicy, { defaultRequestPolicy } from '../../../support/fragments/circulation/request-policy';
+import PatronGroups from '../../../support/fragments/settings/users/patronGroups';
+import Users from '../../../support/fragments/users/users';
+import DefaultUser from '../../../support/fragments/users/userDefaultObjects/defaultUser';
+import ServicePoints from '../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../../support/fragments/users/userEdit';
+import CirculationRules from '../../../support/fragments/circulation/circulation-rules';
+import Checkout from '../../../support/fragments/checkout/checkout';
+import CheckInActions from '../../../support/fragments/check-in-actions/checkInActions';
+import Requests from '../../../support/fragments/requests/requests';
+import NewRequest from '../../../support/fragments/requests/newRequest';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import FixedDueDateSchedules from '../../../support/fragments/circulation/fixedDueDateSchedules';
+import LoanPolicyActions from '../../../support/fragments/circulation/loan-policy';
+
+describe('ui-requests: Request: Edit requests. Make sure that edits are being saved.', () => {
+  // TODO: A client configured to use edge-patron API
```

Comment:
I have some doubts that we need to test here using UI. There is a diagram that shows that we need to call the backend module mod-patron directly. We need to clarify this asking the PO.
https://wiki.folio.org/display/FOLIOtips/Edge+APIs?preview=/25723203/25723207/EdgePatron.png#EdgeAPIs-Diagram.1